2020-05-20-qrexec.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: "Qubes Architecture Next Steps: The New Qrexec Policy System"
+categories: articles
+author: Marta Marczykowska-Górecka
```

Comment:
This should be similar to the previous post in the series: Marek Marczykowski-Górecki, Marta Marczykowska-Górecka